bindata/bootkube/manifests/configmap-initial-aggregator-client-ca.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: initial-aggregator-client-ca
+  namespace: openshift-config
```

Comment:
I think this is logically user-managed.  Technically they could change it and it is not under active management.